spec/cog_cmd/ecs/container/define_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe 'the container-define command' do
+  let(:command_name) { 'container-define' }
+
+  let(:client) { Object.new }
+
+  before do
+    allow(Aws::S3::Client).to receive(:new).and_return(client)
+  end
+
+  it 'defines a new container definition' do
+    ENV['ECS_CONTAINER_DEF_URL'] = 's3://fakebucket/container-definitions'
+    container_def = double(body: double(read: '{"name":"mydef","memory":256,"image":"myimage"}'))
+
+    expect(client).to receive(:put_object)
+    expect(client).to receive(:get_object).and_return(container_def)
+
+    run_command(args: ['mydef', 'myimage'], options: { memory: 256 })
+
+    expect(command).to respond_with({ 'name' => 'mydef',
+                                      'image' => 'myimage',
+                                      'memory' => 256 })
```

Comment:
It is yes. We can discuss changing the cog-rb test stuff at some point. Honestly it seemed fine for now, so I just let it be.